tests-beta/android/MASVS-CRYPTO/MASTG-TEST-0211.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Weak Encryption
```

Comment:
This test is the iOS counterpart of MASTG-TEST-0210. Therefore we should title it the same way.
```suggestion
title: Weak Encryption Algorithms
```